tests/test_lint_recipe.py

```diff
@@ -755,12 +781,13 @@ def test_recipe_v1_test_section_with_recipe(self):
             )
             self.assertIn(expected_message, lints)
 
+            # Note: v1 recipes don't have implicit "run_test.py" support
             with open(os.path.join(recipe_dir, "run_test.py"), "w") as fh:
                 fh.write("# foo")
             lints, hints = linter.lintify_meta_yaml(
                 {}, recipe_dir, recipe_version=1
             )
-            self.assertNotIn(expected_message, lints)
+            self.assertIn(expected_message, lints)
```

Comment:
There is a comment above this one that says

```
# rattler-build supports legacy run_test.py, so we need to test it
```

I cannot tell from the test + comments if we allow implicit `run_test.py` or not.

Reply:
Sorry, removed the comment. That statement is wrong - we don't have this kind of implicit behavior.